index.rb

```diff
@@ -243,6 +270,8 @@ def normalize(text)
                 <strong><a href="<%= h(show.link) %>"><%= normalize(show.title) %></a></strong>
                 <br>
                 <%= normalize(show.description) %>
+                <br>
+                <a href="<%= show.ical_link %>">iCal</a>
```

Comment:
Maybe we should make the date/time text link to this instead?

Reply:
I'm all for that

Reply:
Also here: https://github.com/davishmcclurg/shows/pull/14